target/linux/ath79/dts/qca9563_tplink_archer-c2-v3.dts

```diff
@@ -0,0 +1,195 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca956x.dtsi"
+
+/ {
+	compatible = "tplink,archer-c2-v3", "qca,qca9563";
+	model = "TP-Link Archer C2 Version 3";
```

Comment:
I think this  should be more in line with other model-names, so i would go for "TP-LINK Archer C2 v3"

Reply:
Its seems there are at least two types of device naming schemes in ath79 branch, the one I used (also used for Archer c7, wr1043, wr841, ...), and the one you suggested. Are there any official guidelines on how devices should be named?

Reply:
Indeed, their naming seems to differ. In ar71xx, all TP-Link models were named like i proposed.

I think this is the much more sensible way as TP-Link refers to their devices always as "vX" instead of "Version X". But maybe there are other opinions.